ignite/contrib/handlers/lr_finder.py

```diff
@@ -212,7 +212,7 @@ def plot(
             log_lr: True to plot the learning rate in a logarithmic
                 scale; otherwise, plotted in a linear scale. Default: True.
             display_suggestion: if True, red dot shows the suggested learning rate.
-            filepath: The file name to save the plot to. Default: None.
+            axes: An axes object used for plotting. Default: None.
             kwargs: optional kwargs passed to ``plt.savefig`` if ``filepath`` is provided.
                 matplotlib method `savefig` to save the plot.
```

Comment:
I think I already asked you, please add an example code snippet to this docstring on how to save the figure.